controllers/drcluster_controller_test.go

```diff
@@ -425,6 +425,35 @@ var _ = Describe("DRClusterController", func() {
 		})
 	})
 
+	Context("DRCluster webhook tests/validatiion", func() {
+		Specify("create a drcluster copy for changes", func() {
+			createPolicies()
+			drcluster = drclusters[0].DeepCopy()
+			Expect(k8sClient.Create(context.TODO(), drcluster)).To(Succeed())
+		})
+		When("Update Region after DRCluster is created", func() {
+			It("reports error for Region field update --------------------****************", func() {
+				By("updating region field name")
+				drcluster.Spec.Region = "New Region Updated"
+				Expect(k8sClient.Update(context.TODO(), drcluster)).To(Succeed())
+
+				// get DRCluster
+				//nolint
+				apiReader.Get(context.TODO(), types.NamespacedName{
+					Namespace: drcluster.Namespace,
+					Name:      drcluster.Name,
+				}, drcluster)
+				testLogger.Info("------", "newDRCluster: %w", *drcluster)
+			})
+		})
+		When("deleting a DRCluster and DRPolicy", func() {
+			It("is successful", func() {
+				drpolicyDelete(syncDRPolicy)
+				drclusterDelete(drcluster)
+			})
+		})
+	})
+
```

Comment:
this may be removed, added for testing purposes, will keep it/format as I proceed.